cli/tests/test_windows_installer_artifacts.py

```
@@ -687,6 +696,98 @@ def test_reproducible_launcher_builds_include_final_pe_resources() -> None:
+    harness = tmp_path / "hook-launcher-size.ps1"
+    harness.write_text(
+        "$ErrorActionPreference = 'Stop'\n"
+        "$HookLauncherName = 'defenseclaw-hook-launcher.exe'\n"
+        "$HookLauncherMaxUnsignedBytes = 8MB\n"
+        f"{_builder_function('Assert-HookLauncherArtifact')}\n"
+        "Assert-HookLauncherArtifact $env:DC_TEST_HOOK_LAUNCHER\n",
+        encoding="utf-8",
+    )
+    env = os.environ.copy()
+    env["DC_TEST_HOOK_LAUNCHER"] = str(launcher)
+    accepted = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)